test/airline/AirlineModel.spec.js

```diff
@@ -93,27 +93,6 @@ ava('.getRandomAircraftType() calls ._getRandomAircraftTypeFromFleet() fleet is
     t.true(_getRandomAircraftTypeFromFleetSpy.calledWithExactly(fleetNameMock));
 });
 
-ava('.generateFlightNumber() creates a flightNumber made up of numbers if !flightNumberGeneration.alphaNumeric', (t) => {
```

Comment:
with all the added functionality here, we're going to need a _lot_ more tests.  be we can talk about the _how, when and where_ over the slacks.